app/components/works/keywords_component.html.erb

```diff
@@ -1,5 +1,5 @@
-<div class="keywords" data-controller="keywords" data-action="focusout->keywords#close">
-  <div class="keywords-container <%= 'is-invalid' if errors.present? %>" data-target="keywords.container" data-action="click->keywords#open">
```

Comment:
Remove class because errors are no longer checked and rendered on the server in this partial.